app-backend/db/src/main/scala/CampaignDao.scala

```diff
@@ -154,7 +156,7 @@ object CampaignDao extends Dao[Campaign] with ObjectPermissions[Campaign] {
            INSERT INTO""" ++ tableF ++ fr"(" ++ insertFieldsF ++ fr")" ++
       fr"""SELECT
              uuid_generate_v4(), now(), ${user.id}, name, campaign_type, description, video_link,
-             partner_name, partner_logo, ${id}, continent, ${tagCol}, ${0}, project_statuses""" ++
+             partner_name, partner_logo, ${id}, continent, ${tagCol}, ${0}, project_statuses, ${true}""" ++
```

Comment:
since the `true` is a literal here, would it make more sense to use the Postgres `true`? Alternatively, can I not copy an inactive campaign?